src/util/FormField.js

```diff
@@ -1,266 +1,17 @@
-};
-
-const onTimeChange = (prevDate, fieldName, setFieldValue) => (event) => {
-    const newValue = event.target.value;
-    const date = prevDate ? prevDate : new Date();
-    const dateStr = formatDateObject(date, "YYYY-MM-DD");
-
-    setFieldValue(fieldName, `${dateStr} ${newValue}`);
-};
-
-const FormTimestampField = ({
-    id,
-    label,
-    required,
-    field: { value, onChange, ...field },
-    form: { touched, errors, setFieldValue },
-    ...custom
-}) => {
-    const errorMsg = getFormError(field.name, touched, errors);
-    const date = value && Date.parse(value);
-
-    return (
-        <FormControl error={!!errorMsg}>
-            <TextField
-                id={build(id, "date")}
-                type="date"
-                variant="outlined"
-                label={label}
-                error={!!errorMsg}
-                required={required}
-                value={date ? formatDateObject(date, "YYYY-MM-DD") : ""}
-                onChange={onDateChange(date, field.name, setFieldValue)}
-                {...field}
-                {...custom}
-            />
-            <TextField
-                id={build(id, "time")}
-                type="time"
-                variant="outlined"
-                error={!!errorMsg}
-                required={required}
-                value={date ? formatDateObject(date, "HH:mm:ss") : ""}
-                onChange={onTimeChange(date, field.name, setFieldValue)}
-                {...field}
-                {...custom}
-            />
-            <FormHelperText>{errorMsg}</FormHelperText>
-        </FormControl>
-    );
-};
-
-const FormSelectField = ({
-    id,
-    field: { value, ...field },
-    label,
-    required,
-    form: { touched, errors },
-    children,
-    fullWidth,
-    ...custom
-}) => {
-    const errorMsg = getFormError(field.name, touched, errors);
-    return (
-        <FormControl fullWidth={fullWidth} error={!!errorMsg}>
-            <InputLabel htmlFor={id} required={required}>
-                {label}
-            </InputLabel>
-            <Select id={id} value={value || ""} {...field} {...custom}>
-                {children}
-            </Select>
-            <FormHelperText>{errorMsg}</FormHelperText>
-        </FormControl>
-    );
-};
-
-FormSelectField.defaultProps = {
-    fullWidth: true,
-};
+import FormCheckbox from "./forms/FormCheckbox";
+import FormCheckboxStringValue from "./forms/FormCheckboxStringValue";
+import FormCheckboxTableCell from "./forms/FormCheckboxTableCell";
+import FormIntegerField from "./forms/FormIntegerField";
+import FormMultilineTextField from "./forms/FormMultilineTextField";
+import FormNumberField from "./forms/FormNumberField";
+import FormSearchField from "./forms/FormSearchField";
+import FormSelectField from "./forms/FormSelectField";
+import FormTextField from "./forms/FormTextField";
+import FormTimestampField from "./forms/FormTimestampField";
+import getFormError from "./forms/getFormError";
```

Comment:
I think these can be collapsed with the export below, by using statements like these:
```javascript
export { default as FormCheckbox } from "./forms/FormCheckbox";
export { default as FormCheckboxStringValue } from "./forms/FormCheckboxStringValue";
export { default as FormCheckboxTableCell } from "./forms/FormCheckboxTableCell";
```
...etc.

Reply:
Ah, good call. I'll double-check that does what's expected. It's a lot cleaner-looking.